### ISSUE TITLE
Review Date

### ISSUE THREAD

**Issue by Fikunmi01** [T3xtifyai/Octernship#4]
I submitted a readme file containing link to my presentation then I clicked review date by mistake which extended my submission date does my project gets reviewed?

**Comment by Dannydcodewriter** [comment]
Hi @Fikunmi01 how did you submit your presentation? please I would love to get a response.

**Comment by Fikunmi01** [comment]
Here's a link to the file https://github.com/T3xtifyai/introduction-Fikunmi01/blob/main/monetization-opportunities.md

**Comment by Dannydcodewriter** [comment]
Thank you so much, your presentation is so eye catchy. Which software did you use for the designs?

**Comment by Fikunmi01** [comment]
Thanks, I used canva for that.
When do we get the feedback and where do we get updates on review status.

**Comment by Dannydcodewriter** [comment]
I believe on the due date, which is April 1, we will get a feedback.

**Comment by Fikunmi01** [comment]
Okay thanks, I'm really looking forward to working with this company

**Comment by Dannydcodewriter** [comment]
Same here 
Can i have your contact? We can work on projects together.

**Comment by Fikunmi01** [comment]
Yes, here's my WhatsApp number +2348075332918. I'm a Web developer.

**Comment by Fikunmi01** [comment]
Thanks, I'll be expecting your message

**Comment by AbhishekRP2002** [comment]
My submission date is also showing it's extended. Will it get reviewed or not?

**Comment by Fikunmi01** [comment]
I don't know about that, but I think we're to be reviewed April 1

**Comment by Smartking1** [comment]
Has anyone received any rewiew yet?

**Comment by salimcodes** [comment]
> My submission date is also showing it's extended. Will it get reviewed or not?

Can someone confirm that the deadline has been extended till May 31st? I have not started my assignment yet.
